mbtiles/src/metadata.rs

```diff
@@ -231,30 +232,49 @@ impl Mbtiles {
     }
 }
 
+#[allow(dead_code)]
+pub async fn anonymous_mbtiles(script: &str) -> (Mbtiles, SqliteConnection) {
+    let mbt = Mbtiles::new(":memory:").unwrap();
+    let mut conn = mbt.open().await.unwrap();
+    sqlx::raw_sql(script).execute(&mut conn).await.unwrap();
+    (mbt, conn)
+}
+
+#[allow(dead_code)]
+pub async fn temp_named_mbtiles(
+    file_name: &str,
+    script: &str,
+) -> (Mbtiles, SqliteConnection, PathBuf) {
```

Comment:
easier to return `String` here, esp because this is not really a Path anymore - technically it's a URL

```suggestion
) -> (Mbtiles, SqliteConnection, String) {
```

Reply:
This makes sense, but `mbtiles:patcher` and `mbtiles::copier` expect `PathBuf` as input. The helper function is to be used mostly within these two functions.